lng/english.lng.php

```diff
@@ -2086,5 +2086,9 @@
 $lng['admin']['domain_honorcipherorder'] = 'Honor the (server) cipher order, default <strong>no</strong>';
 $lng['admin']['domain_sessiontickets'] = 'Enable TLS sessiontickets (RFC 5077), default <strong>yes</strong>';
 
-$lng['serversettings']['phpfpm_settings']['custom_config']['title'] = 'Custom Configuration';
-$lng['serversettings']['phpfpm_settings']['custom_config']['description'] = 'Add custom configuration to each PHP-FPM version instance, for example <i>pm.status_path = /status</i> for monitoring. Variables below can be used here.';
+$lng['serversettings']['phpfpm_settings']['restart_note'] = 'Attention: The config won\'t be checked for any errors. If it contains errors, PHP-FPM might not start again!';
```

Comment:
you should fetch and merge the current git version prior to your changes and append your changes at the end of the language files